### PR TITLE
fix: Some --help comments didn't show properly

### PR DESCRIPTION
**Pull request by mateuscomh**
Little fix in some comments when type --help

## Checks
- [ x ] I am happy for maintainers to push small adjustments to this PR, to speed up the review cycle
- [ x ] I have checked that there are no existing pull requests for the same thing

### CODE REVIEW

**Comment by mateuscomh** [comment]
atuin history init-store had the same issue, but i cant  fix that

File: crates/atuin/src/command/client.rs [hunk 1]
```diff
@@ -76,11 +76,12 @@ pub enum Cmd {
     #[command()]
     Doctor,
 
+    /// *Experimental* Enable a background daemon
     #[cfg(feature = "daemon")]
     #[command()]
     Daemon,
 
-    /// Print example configuration
+    /// Print a default atuin configuration (config.toml)
```
Comment (ellie) [inline crates/atuin/src/command/client.rs:84]:
```suggestion
    /// Print the default atuin configuration (config.toml)
```

File: crates/atuin/src/command/client.rs [hunk 2]
```diff
@@ -76,11 +76,12 @@ pub enum Cmd {
     #[command()]
     Doctor,
 
+    /// *Experimental* Enable a background daemon
```
Comment (ellie) [inline crates/atuin/src/command/client.rs:79]:
```suggestion
    /// *Experimental* Start the background daemon
```

**Review by ellie — APPROVED** [review]
Thank you! 😊

**Pull request merged**